### DISCUSSION TITLE
情报：Edge 在标签页右键菜单中新增了开启PiP模式的选项

### DISCUSSION THREAD

**Discussion by Mishasama** [apades/dmMiniPlayer#29] [Show and tell]
确认版本：
Microsoft Edge	128.0.2739.42

这个选项在没有这个扩展的时候激活的是传统的PiP模式；而在有这个扩展的情况下居然直接激活docPiP了， #26 这个BUG也顺带修复掉了。
![image](https://github.com/user-attachments/assets/a1e07070-a8db-46b9-bc4b-7184cd47ff64)

**Comment by apades** [comment]
这个跟我之前 #8 加的有关系，只能说edge那边终于都修了

**Reply by Mishasama** [reply]
比扩展的审核速度还要快，挺好的不是么🙃
